spanner_orm/decorator.py

```diff
@@ -13,18 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Transaction decorators."""
-from typing import Any, Callable
+from typing import Callable, TypeVar
 
 from spanner_orm import api
 
+T = TypeVar('T')
```

Comment:
(nit) A more descriptive name would be nice, especially if you ever add more TypeVars. Maybe `_WrappedFunctionReturnType`?